modules/flowable-bpmn-converter/src/main/java/org/flowable/bpmn/converter/ServiceTaskXMLConverter.java

```diff
@@ -12,20 +12,14 @@
  */
 package org.flowable.bpmn.converter;
 
-import javax.xml.stream.XMLStreamReader;
-import javax.xml.stream.XMLStreamWriter;
-
 import org.apache.commons.lang3.StringUtils;
 import org.flowable.bpmn.converter.export.FieldExtensionExport;
+import org.flowable.bpmn.converter.export.MapExceptionExport;
 import org.flowable.bpmn.converter.util.BpmnXMLUtil;
-import org.flowable.bpmn.model.AbstractFlowableHttpHandler;
-import org.flowable.bpmn.model.BaseElement;
-import org.flowable.bpmn.model.BpmnModel;
-import org.flowable.bpmn.model.CaseServiceTask;
-import org.flowable.bpmn.model.CustomProperty;
-import org.flowable.bpmn.model.HttpServiceTask;
-import org.flowable.bpmn.model.ImplementationType;
-import org.flowable.bpmn.model.ServiceTask;
+import org.flowable.bpmn.model.*;
```

Comment:
Please note the project does not use wildcard imports but explicitly lists each import.  Thanks.